### PR TITLE
Merge for Fix of #2629 and AppZone UI Settings

### LINKED ISSUE #2629: Modelling - add app server network manually
tested with latest dev version v8.4.2 on 2024-11-04
Reproduce by adding a network (e.g. 1.2.3.0/24) in the dialog.

It seems adding works but editing does not!?

- [x] When adding app server networks via UI dialog we get an error "Check constraint violation. new row for relation "owner_network" violates check constraint "owner_network_ip_end_is_host" - even though the values end up in the db correctly as ranges?! 
- [x] When editing the network again, only the first IP is displayed (with /32) - it should actually be displayed as a network again - converting range 1.2.3.0-1.2.3.255 back to 1.2.3.0/24 for displaying.
- [x] Looks like ranges (or networks for that matter) are also not detected as belonging to an area - add unit test for this? 

might already be fixed in develop? need to re-check this!

### PR DESCRIPTION
**Pull request by SolidProgramming**
Closes #2629
UI: Includes turn creation of app zones on/off
UI: Includes make the syntax (AZ) configurable as application zone prefix

### CODE REVIEW

**Comment by SolidProgramming** [comment]
@tpurschke okay...before i push the merge with solved conflicts we should take a look at it together before i break the develop branch completly...

File: roles/lib/files/FWO.Api.Client/FWO.Api.Client.csproj
Comment (tpurschke) [inline roles/lib/files/FWO.Api.Client/FWO.Api.Client.csproj:18]: This should be FWO.Basics

**Review by tpurschke — CHANGES_REQUESTED** [review]

File: roles/database/files/sql/idempotent/fworch-texts.sql
Comment (abarz722) [inline roles/database/files/sql/idempotent/fworch-texts.sql:2270]: Languages mixed up

**Review by abarz722 — APPROVED** [review]

**Review by tpurschke — APPROVED** [review]

**Pull request merged**
